### DISCUSSION TITLE
Call Pod ephemeralcontainers sub-resource operations?

### DISCUSSION THREAD

**Discussion by cronik** [fabric8io/kubernetes-client#4743] [Q&A]
Is it possible to call the [EphemeralContainers Operations](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#-strong-ephemeralcontainers-operations-pod-v1-core-strong-) on a Pod resource? 

```
PATCH /api/v1/namespaces/{namespace}/pods/{name}/ephemeralcontainers
```

It does not appear to be supported directly in the dsl api, but I'm wondering if there is some workaround that might allow me to call the patch operation (maybe by constructing a operations context or something).

Are there plans to support these operations in the dsl? I'd be happy to contribute a PR if it will be welcomed.

**Comment by shawkins** [comment]
> Is it possible to call the [EphemeralContainers Operations](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#-strong-ephemeralcontainers-operations-pod-v1-core-strong-) on a Pod resource?

Currently it is not.

> Are there plans to support these operations in the dsl? I'd be happy to contribute a PR if it will be welcomed.

Not that I have seen.  Please convert this question to an issue and a PR would be fantastic.
